webapp/layouts/default.vue

```diff
@@ -109,11 +109,20 @@
             <!-- logo, hamburger-->
             <ds-flex>
               <ds-flex-item :width="{ base: LOGOS.LOGO_HEADER_WIDTH }" style="margin-right: 20px">
-                <nuxt-link :to="{ name: 'index' }" v-scroll-to="'.main-navigation'">
+                <a
+                  v-if="LOGOS.LOGO_HEADER_CLICK.externalLink"
+                  :href="LOGOS.LOGO_HEADER_CLICK.externalLink"
+                >
```

Comment:
```suggestion
                <a
                  v-if="LOGOS.LOGO_HEADER_CLICK.externalLink"
                  :href="LOGOS.LOGO_HEADER_CLICK.externalLink.url"
                  :target="LOGOS.LOGO_HEADER_CLICK.externalLink.target"
                >
```

Please try if this works if you define `LOGOS.LOGO_HEADER_CLICK.externalLink.url` in `../constants/logos.js`.
And optional:

- `LOGOS.LOGO_HEADER_CLICK.externalLink.target = '_blank'`
  - shall open a new window or tab
- `LOGOS.LOGO_HEADER_CLICK.externalLink.target = ''`
  - shall show the page on the same window and tab and not open a new window or tab

Please implement the same functionality at the logo link for desktop above 👆🏼🙏🏼